docs/adapters/Withdraw.md

```diff
@@ -0,0 +1,57 @@
+## Adapter description and scope
```

Comment:
Flagging that this doc is still a TODO.

Reply:
This is just a documentation template, we don`t really need to provide the adapter information here. It is more like a guide line of the docs we want to see.

Reply:
I thought the doc template is `Template.md` which is a different file? This one is `Withdraw.md` which seems like it needs to be updated to describe the Withdraw adapter.

Reply:
Ah, you are right. I thought we were talking about the template one. This adapter is not ready yet. I will mark it as TODO.